src/legacy/ui/ui_render/bootstrap/template.js.hbs

```diff
@@ -74,6 +74,24 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
       });
     }
 
+    if ('{{appId}}' === 'core') {
+      return load({
+        deps: [
+          {
+            urls: [
+              '{{regularBundlePath}}/kbn-ui-shared-deps/{{sharedDepsFilename}}',
+            ]
+          }
+        ],
+        urls: [
+          '{{regularBundlePath}}/core/core.plugin.js',
```

Comment:
a little weird that it has the `.plugin` suffix. Either we need to customize this or we should probably just change all of the @kbn/optimizer bundles to be `.bundle.js`